src/_lib/sass/src/_popovers.scss

```diff
@@ -14,119 +15,166 @@
     // So reset our font and text properties to avoid inheriting weird values.
     @include reset-text;
     font-size: $font-size-base;
-  
+
     color: get-color("gray", base);
     background-color: $popover-bg;
     background-clip: padding-box;
-    // border: 1px solid $popover-fallback-border-color;
-    // border: 1px solid $popover-border-color;
-    // border-radius: $border-radius-large;
-    // @include box-shadow(0 5px 10px rgba(0,0,0,.2));
-  
+    border: 1px solid $popover-border-color;
+    border-radius: 3px;
+    box-shadow: 0 0 8px rgba(0,0,0,.2);
+    font-family: 'Proxima Nova Regular', Arial, sans-serif;
```

Comment:
I don't think this line is necessary if Analytics has something like the following set:

```
$font-family-sans-serif:  "Helvetica Neue", Helvetica, Arial, sans-serif !default;
$font-family-base:        $font-family-sans-serif !default;
```

Reply:
Removed.